Sources/ConsoleKit/Terminal/Console.swift

```diff
@@ -1,7 +1,15 @@
-import Foundation
-
-#if canImport(Android)
+#if os(macOS)
+import Darwin.C
+#elseif canImport(Glibc)
+import Glibc
+#elseif canImport(Musl)
+import Musl
+#elseif canImport(Android)
 import Android
+#elseif os(WASI)
+import WASILibc
```

Comment:
Are you sure we _need_ these imports? Like if you remove them the app will no longer compile. I'm talking generally about all files, not this file specifically.